engine/src/test/java/com/redhat/cloud/notifications/templates/common/TestAdvisorTemplate.java

```diff
@@ -112,6 +110,7 @@ public void testDailyEmailBodyResolvedOnly() {
 
     @Test
     void testInstantEmailTitleForResolvedRecommendations() {
+        eventTypeDisplayName = "Resolved recommendation";
```

Comment:
This looks unused.

Reply:
Used by parent class [EmailTemplatesRendererHelper](https://github.com/RedHatInsights/notifications-backend/blob/master/engine/src/test/java/com/redhat/cloud/notifications/EmailTemplatesRendererHelper.java#L149).